process/process_openbsd.go

```diff
@@ -227,7 +258,12 @@ func (p *Process) GroupsWithContext(ctx context.Context) ([]int32, error) {
 		return nil, err
 	}
 
-	return k.Groups, nil
+	groups := make([]int32, k.Ngroups)
+	for i := int16(0); i < k.Ngroups; i++ {
```

Comment:
We already have issue #913 on darwin where Ngroups is bigger than the length of Groups. Wouldn't we also have this issue here?

Reply:
This fix got here by accident from a neighboring branch. Removing it from here, since it belongs to #964